pkg/helpers/fs.go

```diff
@@ -96,3 +96,13 @@ func RemoveAll(path string) error {
 	}
 	return me.ErrorOrNil()
 }
+
+// InitDir creates a directory if it does not exist and make sure it has the right permissions
+func InitDir(dir string, perm os.FileMode) error {
+	if err := os.MkdirAll(dir, perm); err != nil {
+		return fmt.Errorf("unable to create directory %s: %w", dir, err)
```

Comment:
i see no reason to wrap this. it is redundant

```suggestion
		return err
```